src/components/include/test/policy/policy_external/policy/mock_policy_manager.h

```diff
@@ -148,6 +148,51 @@ class MockPolicyManager : public PolicyManager {
       StatusNotifier(
           const std::string& application_id,
           const rpc::policy_table_interface_base::AppHmiTypes& hmi_types));
+#ifdef SDL_REMOTE_CONTROL
+  MOCK_METHOD2(SetDefaultHmiTypes,
+               void(const std::string& application_id,
+                    const std::vector<int>& hmi_types));
+  MOCK_METHOD2(GetHMITypes,
+               bool(const std::string& application_id,
+                    std::vector<int>* app_types));
+  MOCK_METHOD6(CheckAccess,
```

Comment:
@AByzhynar probably you should update this mock. For example here `MOCK_METHOD5`